utils/run_in_ga.sh

```diff
@@ -14,12 +14,12 @@ rm -rf /opt/daos/prereq/release/spdk
 $SCONS PREFIX=/opt/daos --build-deps=yes --deps-only
 echo ::endgroup::
 
-echo ::group::Prepare stack analyzer
-$SCONS --jobs 10 server
+echo ::group::Stack analyzer output (post build)
```

Comment:
(lint) '(' is invalid here. Did you forget to escape it? [SC1036]
(lint) Parsing stopped here. Invalid use of parentheses? [SC1088]